fractal_tasks_core/tasks/illumination_correction.py

```diff
@@ -182,13 +173,18 @@ def illumination_correction(
             raise ValueError(f"{well=}, {new_well=}")
         zarrurl_new = (Path(output_path) / new_component).as_posix()
 
+    # Read some parameters from metadata
+    ngff_image_meta = load_NgffImageMeta(zarrurl_old)
+    num_levels = ngff_image_meta.num_levels
+    coarsening_xy = ngff_image_meta.coarsening_xy
+
     t_start = time.perf_counter()
     logger.info("Start illumination_correction")
     logger.info(f"  {overwrite_input=}")
     logger.info(f"  {zarrurl_old=}")
     logger.info(f"  {zarrurl_new=}")
 
-    # Read channels from .zattrs
+    # Read channels from .zattrs # FIXME replace with NgffImageMeta?
```

Comment:
They should be available in the NGFF model now, no?

Reply:
Deferred to:
* https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/540